### ISSUE TITLE
bug: GFortran version on WSL not parsed by linter REGEX

### ISSUE THREAD

**Issue by Hongrui-Yu** [fortran-lang/vscode-fortran-support#759]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Setting Linter compiler to "gfortran", open any support Fortran source file and open log in VS Code. No error message is shown if "ifort" is selected as linter compiler.

### Screenshots

![image](https://user-images.githubusercontent.com/30503849/203436370-266ab8a0-36a5-4f7b-b2e4-1219e1eb0e7b.png)

### Expected Behaviour

No error message. For example, if "ifort" is chosen as linter compiler:

![image](https://user-images.githubusercontent.com/30503849/203437277-665035ec-6826-4f50-8ae7-64f17c60ddfe.png)

### Version of Modern Fortran

v3.4.0

### Version of Visual Studio Code

v1.73.1

### Platform and Architecture

Windows with WSL

### Additional Information

Despite the error message, linting seems to work fine. Settings used for Modern Fortran Extension and fortls

```
    "fortran.fortls.maxLineLength": 132,
    "fortran.fortls.path": "/home/yuhongrui/.local/bin/fortls",
    "fortran.linter.modOutput": "${workspaceFolder}/fortran_mods",
    "fortran.linter.includePaths": [
        "/usr/lib/x86_64-linux-gnu/openmpi/include",
        "/usr/lib/x86_64-linux-gnu/openmpi/lib",
        "/usr/lib/x86_64-linux-gnu/openmpi/lib",
        "/home/yuhongrui/petsc/arch-linux-c-debug/include",
        "/home/yuhongrui/petsc/include",
        "${workspaceFolder}/fortran_mods"
    ],
    "fortran.linter.extraArgs": [
        "-Wl,-rpath,/home/yuhongrui/petsc/arch-linux-c-debug/lib",
        "-L/home/yuhongrui/petsc/arch-linux-c-debug/lib",
        "-L/usr/lib/x86_64-linux-gnu/openmpi/lib",
        "-fPIC",
        "-w",
        "-ffree-line-length-none",
        "-ffree-line-length-0",
        "-Wno-lto-type-mismatch",
        "-Wno-unused-dummy-argument",
        "-pthread",
        "-g",
        "-O0",
        "-lpetsc",
        "-lm",
        "-lmpi_usempif08",
        "-lmpi_usempi_ignore_tkr",
        "-lmpi_mpifh",
        "-lmpi"
    ],
    "fortran.fortls.preprocessor.directories": [
        "$HOME/petsc/include/petsc/finclude"
    ],
    "fortran.fortls.directories": [
        "$HOME/petsc/include/petsc/finclude"
    ],
    "fortran.fortls.suffixes": [
        ".h"
    ],
    "fortran.fortls.preprocessor.definitions": {
        "Mat": "type(tMat)",
        "Vec": "type(tVec)",
        "SETERRA": "call PetscError(c,ierr,0,s); call MPIU_Abort(c,ierr)",
        "PetscErrorCode": "type(PetscErrorCode)",
        "PetscMPIInt": "type(PetscMPIInt)",
        "SNES": "type(tSNES)",
        "KSP": "type(tKSP)",
        "PC": "type(tPC)",
        "SNESLineSearch":"type(SNESLineSearch)",
        "SNESConvergedReason":"type(SNESConvergedReason)"

    },
    "fortran.fortls.excludeDirectories": [
        "$HOME/.vscode-server"
    ],
    "fortran.linter.initialize": false
```

**Comment by gnikit** [comment]
What is the output of `gfortran --version`?

**Comment by Hongrui-Yu** [comment]
GNU Fortran (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

**Comment by gnikit** [comment]
Thanks, I'll have a look and adjust the regular expression parsing the gfortran version

**Comment by gnikit** [comment]
@Hongrui-Yu Fixed in PR #761 will be available for use in tomorrows nightly release.
